src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -0,0 +1,132 @@
+package uk.gov.companieshouse.service;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.when;
+
+import org.junit.Rule;
```

Comment:
Bit shame you did not use jUnit 5 with bit nicer API but no reason to put extra an effort now in the light of our deadline.

Reply:
Now using junit 5 :-)